contracts/test/permission/permission-condition.ts

```diff
@@ -1,21 +1,23 @@
 import {
-  IERC165__factory,
   IPermissionCondition__factory,
   IProtocolVersion__factory,
   PermissionConditionMock,
   PermissionConditionMock__factory,
 } from '../../typechain';
+import {erc165ComplianceTests} from '../helpers';
 import {getInterfaceId} from '@aragon/osx-commons-sdk';
+import {SignerWithAddress} from '@nomiclabs/hardhat-ethers/signers';
 import {expect} from 'chai';
 import {ethers} from 'hardhat';
 
 // TODO
```

Comment:
either remove or give a sentence what needs to be done

Reply:
Done in https://github.com/aragon/osx-commons/pull/40/commits/a7e7a6fc8d7c667af246f97e98c5ed017fc13a29